docs/content/docs/install/metrics.md

```diff
@@ -10,6 +10,7 @@ The metrics for pipelines-as-code can be accessed through the `pipelines-as-code
 pipelines-as-code supports various exporters, such as Prometheus, Google Stackdriver, and more.
 You can configure these exporters by referring to the [observability configuration](../config/config-observability.yaml).
 
-|  Name | Type    | Description                                         |
-| ---------- |---------|-----------------------------------------------------|
-| `pipelines_as_code_pipelinerun_count` | Counter | Number of pipelineruns created by pipelines-as-code |
+| Name                                                 | Type    | Description                                                        |
+|------------------------------------------------------|---------|--------------------------------------------------------------------|
+| `pipelines_as_code_pipelinerun_count`                | Counter | Number of pipelineruns created by pipelines-as-code                |
+| `pipelines_as_code_pipelinerun_duration_seconds_sum` | Counter | Number of seconds all pipelineruns have taken in pipelines-as-code |
```

Comment:
is it the sum taken of all pipelinerun install?

what is the usefulness of this ? was this coming from a customer/user request?

Reply:
@chmouel console team wants total number of pipelineruns metric (which we already have) and sum of all duration so that they can show average time taken by pipelinerun in specified time range.

slack thread: https://redhat-internal.slack.com/archives/CG5GV6CJD/p1726742648504399